### PR TITLE
feature/Adding postgres container

### PR DESCRIPTION
**Pull request by KirylKiryanchykau**
Add postgres container

### CODE REVIEW

**Comment by Deffiss** [comment]
It looks good, but could you please update Readme.md as well? Also right now publishing is a manual process, so once we merge this I'll do the nuget upgrade.

**Comment by KirylKiryanchykau** [comment]
Sure, updated

File: test/TestEnvironment.Docker.Tests/TestEnvironment.Docker.Tests.csproj [hunk 1]
```diff
@@ -24,6 +24,7 @@
     <ProjectReference Include="..\..\src\TestEnvironment.Docker.Containers.Mongo\TestEnvironment.Docker.Containers.Mongo.csproj" />
     <ProjectReference Include="..\..\src\TestEnvironment.Docker.Containers.Mssql\TestEnvironment.Docker.Containers.Mssql.csproj" />
     <ProjectReference Include="..\..\src\TestEnvironment.Docker\TestEnvironment.Docker.csproj" />
+    <ProjectReference Include="..\..\TestEnvironment.Docker.Containers.Postgres\TestEnvironment.Docker.Containers.Postgres.csproj" />
```
Comment (Hellevar) [inline test/TestEnvironment.Docker.Tests/TestEnvironment.Docker.Tests.csproj]: Something wrong with project location, check Mssql project path above for example

Comment (KirylKiryanchykau) [reply test/TestEnvironment.Docker.Tests/TestEnvironment.Docker.Tests.csproj]: Project path corrected

**Pull request merged**